src/App.js

```diff
@@ -30,14 +30,17 @@ class App extends Component {
   handleSubmit(event) {
     if(!quotes.includes(this.state.value)){
       quotes.push(this.state.value);
-      this.setState({ quote: this.getQuote()});
+      this.renderNewQuote();
 
       console.log(this.state.quote )
     }
     this.persistQuotes();
     event.preventDefault();
   }
   
+  renderNewQuote(event) {
```

Comment:
I'd probably call this `fetchAnotherQuote` or `updateQuote` or something.  The rendering is done by another method (`render` at the moment).  Obviously this method updates the state, which will force another call to `render`, but the distinction will begin to matter more as you get into redux/actions/reducers